core/hydromet_stratified.py

```diff
@@ -353,7 +351,7 @@ def Scenarios_Avg_S_Median_S(df_weights_runoff: pd.DataFrame, mu: float, GEV_par
                           S_limit) for Q1 in Runoff_Q]
     R_Avg_S = [1.0/2.0*(Q+np.sqrt(Q)*np.sqrt(Q+4.0*S)+2.0*S*mu) for Q, S in zip(Runoff_Q, Avg_S_list)]
     Median_S_list = [Median_S(Q1, mu, GEV_parameters, PMP, partition_avg, Delta_P, alpha, beta, S_limit, error_PQ, 
-                              [(0.25, S_limit)], 1.5).x[0] for Q1 in Runoff_Q]
+                              [(0.25, S_limit)], (0+S_limit)/3).x[0] for Q1 in Runoff_Q]
```

Comment:
Not sure why the inititial value was changed to (0+S_limit)/3 but I will defer

Reply:
Fixed, changed so that the search for the maximum starts in a more appropriate location.